public/data/CanisCoder.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Anand Kumar Maurya",
+  "bio": "Open Source Enthusiast | MERN Stack Developer | Competitive Coder | CSE student at @AITM ",
```

Comment:
There's an extra space here.
```suggestion
  "bio": "Open Source Enthusiast | MERN Stack Developer | Competitive Coder | CSE student at @AITM",
```

Reply:
I think the GitHub format action will take care of it!